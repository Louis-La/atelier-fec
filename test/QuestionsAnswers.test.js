import React from 'react';

import { shallow } from 'enzyme';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import ProductOverview from '../client/src/components/productDetails/products.jsx';

Enzyme.configure({ adapter: new Adapter() });

describe('Basic test', () => {
  it('adds 1 + 2 to equal 3', () => {
    const sum = (a, b) => {
      return a + b;
    };
    expect(sum(1, 2)).toBe(3);
  });
});

describe('ProductOverview', () => {
  it('should be true that ProductOverview exists', () => {
    const wrapper = shallow(<ProductOverview />);
    expect(wrapper.find(ProductOverview)).toBeTruthy();
  });
});
