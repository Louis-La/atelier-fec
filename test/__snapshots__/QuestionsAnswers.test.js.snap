// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Each Component renders correctly AddToCart renders correctly 1`] = `ShallowWrapper {}`;

exports[`Each Component renders correctly AddToCart renders correctly 2`] = `ShallowWrapper {}`;

exports[`Each Component renders correctly AddToCart renders correctly 3`] = `ShallowWrapper {}`;

exports[`Each Component renders correctly AddToCart renders correctly 4`] = `ShallowWrapper {}`;

exports[`Each Component renders correctly AddToCart renders correctly 5`] = `ShallowWrapper {}`;

exports[`Each Component renders correctly AddToCart renders correctly 6`] = `ShallowWrapper {}`;

exports[`Each Component renders correctly AddToCart renders correctly 7`] = `ShallowWrapper {}`;

exports[`Each Component renders correctly AddToCart renders correctly 8`] = `ShallowWrapper {}`;

exports[`Each Component renders correctly ProductInformation renders correctly 1`] = `ShallowWrapper {}`;

exports[`Each Component renders correctly StyleSelector renders correctly 1`] = `ShallowWrapper {}`;
